.Lib9c.Tests/Action/RewardGoldTest.cs

```diff
@@ -22,7 +22,8 @@ public RewardGoldTest()
         {
             var sheets = TableSheetsImporter.ImportSheets();
             sheets[nameof(CharacterSheet)] =
-                "id,_name,size_type,elemental_type,hp,atk,def,cri,hit,spd,lv_hp,lv_atk,lv_def,lv_cri,lv_hit,lv_spd,attack_range,run_speed\n100010,전사,S,0,300,20,10,10,90,70,12,0.8,0.4,0,3.6,2.8,2,3";
+                @"id,_name,size_type,elemental_type,hp,atk,def,cri,hit,spd,lv_hp,lv_atk,lv_def,lv_cri,lv_hit,lv_spd,attack_range,run_speed
+                  100010,전사,S,0,300,20,10,10,90,70,12,0.8,0.4,0,3.6,2.8,2,3";
```

Comment:
Are the leftmost space characters intended?

Reply:
I was just trying to indent the string literal, but I just found the actual assigned value is not intended. Instead I will indent it using `string.Join()`!